src/components/input/demoBasicUsage/index.html

```diff
@@ -53,7 +53,7 @@
         </md-input-container>
         <md-input-container flex>
           <label>Postal Code</label>
-          <input ng-model="user.postalCode">
+          <input ng-model="user.postalCode" placeholder="12345">
```

Comment:
The styling is added dynamically based on whether they include both the `label` and the `placeholder` which are standard attributes. I think the idea was to make the inputs as close to a standard HTML input as possible.

It's all done/decided in the bottom of `input.js` in the `function placeholderDirective()` and documented in the input docs.

Is this what you were asking?